LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
@rieske perhaps we can link to this in the README?

Reply:
I *think* GitHub will pick this up and start showing it on the right when the project is opened automatically.

![image](https://user-images.githubusercontent.com/5560673/157662265-3cadaf7e-25ab-43fe-bcf2-39861a977cc8.png)